requirements/edx/development.txt

```diff
@@ -42,7 +42,6 @@ git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.1.6#egg=xblock-d
 -e git+https://github.com/edx-solutions/xblock-google-drive.git@138e6fa0bf3a2013e904a085b9fed77dab7f3f21#egg=xblock-google-drive
 git+https://github.com/open-craft/xblock-poll@add89e14558c30f3c8dc7431e5cd6536fff6d941#egg=xblock-poll==1.5.1
 -e git+https://github.com/raccoongang/xblock-pdf.git@sgab-v.1.0.1#egg=xblock-pdf
-git+ssh://git@github.com/edly-io/edly-panel-edx-app.git#egg=edly-panel-app
```

Comment:
was this causing any issue, so we can remove this from our other forked repos.

Reply:
development.txt is used while devstack setup, if we don't have access to edly repos then it will hinder the process of paver update_assests.
As we are not using this app anymore so I thought it would be better if we remove it from here as well to help newcomers.

Reply:
thanks, I would like to make sure that nothing breaks with this change.

Reply:
I have tested on stage, it's working fine.